lib/pixel-mapper.cc

```diff
@@ -217,6 +217,54 @@ class UArrangementMapper : public PixelMapper {
   int parallel_;
 };
 
+
+
+class VerticalMapper : public PixelMapper {
+public:
+  VerticalMapper() {}
+
+  virtual const char *GetName() const { return "V-mapper"; }
+
+  virtual bool SetParameters(int chain, int parallel, const char *param) {
+    if (chain < 2) {
```

Comment:
Why this limitation ? With a chain of one it would be possible to make a long panel (e.g. for a runtext) that is fast to update (because it then would be three short segments, that make use of parallel feeding)

```
[O < I] [O < I] [O < I]
     |       |       |
     ch3     ch2      ch1
```

Looking at the code below, it _should_ be not a problem (we never divide by (chain_ - 1) for instance).

Reply:
@hzeller  your are absolutely correct that chains of 1 would both work, and make sense. I updated that.
You are also correct that I got the ASCII art reverted (not counting left-right which is a bit tricky as it depends on whether you are showing the panels with pixels facing you or with the connectors facing you, but I think that one isn't too important, the up/down one is; that said I changed both).